docs/fastdds/docker/docker_demo/fast_dds_helloworld.rst

```diff
@@ -0,0 +1,26 @@
+.. include:: ../../../03-exports/aliases.include
+.. include:: ../../../03-exports/aliases-api.include
+.. include:: ../../../03-exports/roles.include
+
+.. _fast_dds_hello_world:
+
+Fast DDS Hello World
+====================
+
+This Docker image contains **eProsima Fast DDS** bundled with a simple Hello World application that can be used as a quick 
+demonstration of the communication capabilities between a Publisher and a Subscriber using Fast DDS. 
+
+To load this image into your Docker repository, from a terminal run
+
+.. code-block:: bash
+
+   $ docker load -i ubuntu-fast-dds-helloworld:<FastDDS-Version>.tar
+
+You can run this Docker container as follows
+
+.. code-block:: bash
+
+   $ docker run -it ubuntu-fast-dds-helloworld:<FastDDS-Version>
+
+This will perform a Publisher/Subscriber match and start sending samples. This example is not constrained to the 
+current instance. It's possible to run several instances of this container to check the communication between them.
```

Comment:
This should be moved to the Fast DDS Suite section to complete the HelloWorld part. The rest can be removed because this Docker is not going to be generated/released.